docs/Guide.Mocking.md

```diff
@@ -36,6 +36,19 @@ This replacement mechanism provides a lot of flexibility to change implementatio
 
     ```
 
+    or use the new format for metro configuration for RN >= 0.57 (or if you've metro >= 0.43)
```

Comment:
"if you've" is not a valid English phrase when "have" has a possessive meaning.

Reply:
It doesn't really matter. It will be squashed and merged at the end. Also, please capitalize "Metro".

I also think that going forward, we should have the new configuration be first, rather than "or use this new one".

Reply:
Ok, I'll make all the corrections including changing the new configuration to be first

Reply:
Thanks. Let's wait for @rotemmiz to review